dozuki.js

```diff
@@ -3,21 +3,24 @@
    function Dozuki(url, http) {
       baseUrl = url + "/api/2.0/";
       this.guides = {
-         get: function(guideid, langid, params) {
+         get: function(guideid, langid, params, headers) {
+            headers = headers || {};
```

Comment:
Although, I do see a case where you would want to add different headers depending on the API call you are making.  But that's not really important right now since there's only one function.